### PR TITLE
refactor(tooltip): changed and improved tooltips component

### PR DESCRIPTION
**Pull request by ThomBos**
**What this PR does** 📖
Improves tooltip performances and display and makes tooltips code easier to change

**Which issue(s) this PR fixes** 🔨
AP-1595

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

**Comment by molimauro** [comment]
I saw the chatbar controls does not have the `cursor: pointer` attr ;)

**Comment by molimauro** [comment]
Hey @ThomBos thanks! Now i have only 2 doubts, one is regarding the appearing effect of the tooltip, and the other is the arrow of the tooltip that we didn't have previously. Can you talk with Liz about these and let us know? Thanks!

**Comment by ThomBos** [comment]
> Hey @ThomBos thanks! Now i have only 2 doubts, one is regarding the appearing effect of the tooltip, and the other is the arrow of the tooltip that we didn't have previously. Can you talk with Liz about these and let us know if we have to keep them? Thanks!

I had a talk with Liz, so we are now keeping both animations and arrows!! 🔥

File: components/interactables/Button/Button.html [hunk 1]
```diff
@@ -5,8 +5,7 @@
   :type="htmlType"
   ref="button"
 >
-  <!-- {{$attrs['data-tooltip']}} -->
-  <UiGlowingCursorArea :hide="$attrs['data-tooltip'] ? true : false">
+  <UiGlowingCursorArea :hide="$attrs['data-popper-shown'] ? true : false">
```
Comment (aewing) [inline components/interactables/Button/Button.html:8]: I think you could shorten this with `:hide="!!$attrs['data-popper-shown']"`

File: assets/styles/base.less [hunk 2]
```diff
@@ -709,24 +709,63 @@ video {
...
-    &:extend(.blur-less);
-    font-family: @primary-font !important;
-    color: @secondary-text !important;
-    &:extend(.fourth-layer);
+li .is-active,
+li.active,
+input.date-input {
+  &:extend(.background-flair);
+  &:extend(.glow-flair);
+  border: none !important;
```
Comment (aewing) [inline assets/styles/base.less:720]: This CSS rule is a little concerning at first glance. Are we sure that we want to style any item that has the `is-active` class inside of an `li`, any `li` with the `active class`, and all `date-input` with a `border: none !important`?

File: assets/styles/base.less [hunk 3]
```diff
@@ -709,24 +709,63 @@ video {
...
+}
+
+// ------------- Ui Elements-------------
+
+* {
+  border-color: var(--flair-color);
```
Comment (aewing) [inline assets/styles/base.less:726]: This feels a little far-reaching also

File: components/interactables/Button/Button.html (hunk 1 above)
Comment (ThomBos) [reply components/interactables/Button/Button.html:8]: Done! I didn't know it was possible to do that, thanks 😄

File: assets/styles/base.less (hunk 3 above)
Comment (molimauro) [reply assets/styles/base.less:726]: Totally agree. @ThomBos why this rule in this PR?

Comment (ThomBos) [reply assets/styles/base.less:726]: this isn't on this pr, i removed the class above ".has-tooltip"

Comment (molimauro) [reply assets/styles/base.less:726]: ok ill open a ticket for this one and the other and we'll address those there

Comment (molimauro) [reply assets/styles/base.less:726]: https://satellite-im.atlassian.net/browse/AP-1635

**Review by molimauro — APPROVED** [review]

**Review by stavares843 — APPROVED** [review]

**Pull request merged**